opm/parser/eclipse/EclipseState/Grid/EclipseGrid.hpp

```diff
@@ -59,10 +57,12 @@ namespace Opm {
           These constructors will make a copy of the src grid, with
           zcorn and or actnum have been adjustments.
         */
-        EclipseGrid(const EclipseGrid& src, const double* zcorn , const std::vector<int>& actnum);
-        EclipseGrid(const EclipseGrid& src, const std::vector<double>& zcorn , const std::vector<int>& actnum);
+        EclipseGrid(const EclipseGrid& src) = default;
         EclipseGrid(const EclipseGrid& src, const std::vector<int>& actnum);
+        EclipseGrid(const EclipseGrid& src, const std::vector<double>& zcorn, const std::vector<int>& actnum);
 
+        EclipseGrid(const EclipseGrid& src, const double* zcorn, const std::vector<int>& actnum);
```

Comment:
I have looked into this and I have found that opm-grid is using both constructors

in GridHelpers.cpp:34
```EclipseGrid createEclipseGrid(const Dune::CpGrid& grid, const EclipseGrid& inputGrid)```
...
```return Opm::EclipseGrid( inputGrid, grid.zcornData( ).data() , updatedACTNUM );```

in GridHelpers.cpp:343
```Opm::EclipseGrid createEclipseGrid(const UnstructuredGrid& grid, const Opm::EclipseGrid& inputGrid ) {```
...
```return Opm::EclipseGrid( inputGrid, grid.zcorn, updatedACTNUM );```

If I should keep only one in EclipseGrid I will have to update GridHelpers.cpp in opm-grid. If so, I guess it would be easier to keep the one with pointer double* zcorn, and change 

GridHelpers.cpp:47 

from 

```return Opm::EclipseGrid( inputGrid, grid.zcornData( ) , updatedACTNUM );```

to 

```return Opm::EclipseGrid( inputGrid, grid.zcornData( ).data() , updatedACTNUM );```
 
What do you recommend ?